src/util/data-gov-api.ts

```diff
@@ -0,0 +1,40 @@
+import axios from 'axios';
+
+interface Response {
```

Comment:
nit: accept a generic type parameter to use as the record type

```suggestion
interface Response<TRecord> {
```